### DISCUSSION TITLE
Facebook page

### DISCUSSION THREAD

**Discussion by TeacherRabbit123** [netless-io/flat#1842] [General] [answered]
I want to let other teachers know this. Is it okay to make a Facebook page about it?

**Comment by hyrious** [comment, marked as answer]
Thank you for trying Flat. We are happy to see more users use & spread it.

**Reply by TeacherRabbit123** [reply]
Thanks a lot! I will post the link of the Facebook group after I create it. I think we can have more testers there and it will be better to help make this software better in the future.

**Reply by TeacherRabbit123** [reply]
Here is the link to the Facebook page that I have created, sir. 
[https://www.facebook.com/groups/733100225139405](url)
